src/apps/settings/src/app/views/Robots/Robots.js

```diff
@@ -179,7 +180,7 @@ export default connect(state => {
           </h2>
           <iframe
             className={styles.Iframe}
-            src={`${robotURL}?q=${Math.random()
+            src={`https://${robotURL.slice(7)}?q=${Math.random()
```

Comment:
I would move this up into the functional component body so we can include a comment which explains what is happening and why it needs to be done.